spec/flow/common_spec.rb

```diff
@@ -6,87 +6,167 @@
+    expect(capture.status.value).to eq('succeeded')
+  end
 
-    # we allready deleted transaction, this should fail
-    response = gateway.void(nil, authorization_key)
-    expect(response.success?).to be_falsey
+  it 'voids (cancels) the authorization' do
+    authorization = gateway.flow_get_authorization order_number: test_order_number
+
+    expect(authorization.key.include?('aut-')).to be(true)
+
+    response = gateway.void 0.1, authorization.key, currency: 'USD'
+    expect(response.success?).to be_truthy
+    expect(response.params['response'].key.include?('rev-')).to be(true)
   end
 
-  it 'refunds the transaction by capture id' do
-    response   = gateway.purchase(amount, credit_card, currency: 'USD')
-    capture_id = response.params['response'].id
+  it 'refunds the transaction by authorization_id' do
+    authorization = gateway.flow_get_authorization order_number: test_order_number
+    response      = gateway.capture 0.1, authorization.key
+    capture       = response.params['response']
 
-    expect(capture_id.include?('cap-')).to be_truthy
+    expect(capture.nil?).to be(false)
+    expect(capture.key.include?('cap-')).to be_truthy
+    expect(capture.authorization.key).to eq(authorization.key)
+    expect(capture.status.value).to eq('succeeded')
 
-    response = gateway.refund(nil, capture_id)
-    expect(response.id.include?('ref-')).to be_truthy
+    response = gateway.refund 0.1, capture.id, currency: 'USD'
+    expect(response.success?).to be_truthy
+    expect(response.params['response'].key.include?('ref-')).to be(true)
 
-    expect{ gateway.refund(nil, capture_id) }.to raise_error(Io::Flow::V0::HttpClient::ServerError)
+    bad_response = gateway.refund(nil, capture.id)
+    expect(bad_response.success?).to be(false)
   end
 
-  it 'refunds the transaction by authorization key' do
-    auth_response = gateway.authorize amount, credit_card, currency: 'USD'
-    gateway.capture amount, auth_response.authorization
+  # it 'checks create order, authorize and capture (if not in review) ability' do
+  #   # Authorize $10 from the credit card
+  #   # auth_response = gateway.authorize(amount, credit_card, currency: 'USD')#, order_number: 'ord-28bc0cc14db6433d8bdfa51ff6878511')
 
-    auth_id = auth_response.params['response'].id
-    expect(auth_id.include?('aut-')).to be_truthy
+  #   expect(auth_response.message).to eq "Flow authorize - Success"
 
-    response = gateway.refund(nil, nil, authorization_id: auth_id)
-    expect(response.id.include?('ref-')).to be_truthy
-  end
+  #   # Capture $10 from the credit card
+  #   capture_response = gateway.capture(amount, auth_response.authorization)
 
-  it 'checks storage of credit card' do
-    response = gateway.store(credit_card)
-    expect(response.success?).to be_truthy
-    expect(response.params['token'].length).to eq(64)
-    expect(response.params['response'].id.include?('crd-')).to be_truthy
-  end
+  #   expect(capture_response.success?).to be_truthy
```

Comment:
any particular reason the remaining code is commented out?  suggest we either uncomment or remove